packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -168,10 +172,14 @@ export class Popup extends React.Component<IPopupProps, IPopupState> {
   };
 }
 
-function defaultFocusRestorer(options: { originalElement?: HTMLElement | Window; containsFocus: boolean }) {
-  const { originalElement, containsFocus } = options;
+function defaultFocusRestorer(options: {
+  originalElement?: HTMLElement | Window;
+  containsFocus: boolean;
+  documentContainsFocus: boolean;
+}) {
+  const { originalElement, containsFocus, documentContainsFocus } = options;
 
-  if (originalElement && containsFocus && originalElement !== window) {
+  if (originalElement && containsFocus && documentContainsFocus && originalElement !== window) {
```

Comment:
Let's leave the `documentContainsFocus` check out for now as it might break existing scenarios.

Reply:
Users can chose to check documentContainsFocus if they want in their own override.